.github/workflows/python-ci-wheel.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install twine
+          python -m pip install importlib_resources twine
```

Comment:
The latest version that we currently "support" is 3.7, and `importlib.resources` seems to be available there?

Reply:
Oh, sorry -- I see that it's installing it in the CI...

Reply:
The files package is available from 3.9 onwards: https://docs.python.org/3.9/library/importlib.html#module-importlib.resources

Reply:
btw, pypi.org still says "Python >=3.5". Should we update this?